### ISSUE TITLE
Need help: How to "subscribe" to a feed?

### ISSUE THREAD

**Issue by AndreasGassmann** [qaap/rpc-websockets#26]
I'm trying to use the this library to get market/trading data from HitBTC. (e.g. subscribing to ticker: https://api.hitbtc.com/#subscribe-to-ticker)

The problem is that I'm not 100% sure how I have to do that.

When I do the following it just prints "true" and doesn't print anything else.

```javascript
ws.on('open', function() {
  ws.call('subscribeTicker', {
    "symbol": "ETHBTC", 
  }).then(function(result) {
    console.log(result);
  }).catch(console.log);
});
```

I dug into the code and saw that the library receives the updates from the server after `ws.call('subscribeTicker', ...)`, but the messages don't get forwarded. When I add a `console.log(message)` here (https://github.com/qaap/rpc-websockets/blob/master/src/lib/client.js#L210) I get this response: 

```javascript
{ jsonrpc: '2.0',
  method: 'ticker',
  params:
   { ask: '0.102482',
     bid: '0.102339',
     last: '0.102483',
     open: '0.111648',
     low: '0.096804',
     high: '0.113327',
     volume: '70636.904',
     volumeQuote: '7539.892416786',
     timestamp: '2018-02-02T22:16:24.866Z',
     symbol: 'ETHBTC' } }
{ channel: 'ticker',
  type: 'update',
  data:
   { ask: '0.102482',
     bid: '0.102341',
     last: '0.102483',
     open: '0.111648',
     low: '0.096804',
     high: '0.113327',
     volume: '70636.904',
     volumeQuote: '7539.892416786',
     timestamp: '2018-02-02T22:16:24.966Z',
     symbol: 'ETHBTC' } }
```
There is no `notification` property nor an ID and it doesn't get handled. I hacked something together to expose an additional "callback" function and I was able to use that and it works fine. But that's an ugly hack I would like to avoid.

So my question is: Am I missing a feature of this library? Or did they simply implement their API in a very weird way that is not supported by this library?

Thanks for your help.

**Comment by mkozjak** [comment]
@AndreasGassmann, thanks for reporting this. I believe this is a bug in our library. I'm trying to patch it and will release it asap.

**Comment by mkozjak** [comment]
@AndreasGassmann: This is fixed in 3.6.2.

You would use it like this:
```js
ws.on("open", function() {
  ws.call("subscribeTicker", { "symbol": "ETHBTC" })
  .then(function() {
    ws.on("ticker", function(msg) {
      console.log("tick!!!", msg)
    })
  })
  .catch(function(error) {
    console.log("error subscribing:", error)
  })
})
```

`ws.on` is an event handler, and since they expose `ticker` as their notification, it will end up there.
